manager-bundle/skeleton/config/security.yaml

```diff
@@ -17,6 +17,13 @@ security:
             user_checker: contao.security.user_checker
             switch_user: true
             login_throttling: ~
+            webauthn:
```

Comment:
We have to add this to the `README.md` and `core-bundle/tests/Functional/app/config/security.yaml` files, too.